download.tt

```diff
@@ -0,0 +1,339 @@
+      images onto a blank CD/DVD disk, or <a href="https://nixos.org/nixos/manual/index.html#sec-booting-from-usb">by
+        copying it onto a USB stick</a>. For <strong>installation
+        instructions</strong>, please see the <a href="[%nixosManual%]">manual</a>.</p>
+
+    <div class="alert alert-block">
+      Please note that NixOS at the moment lacks a nice, user-friendly
+      graphical installer. Therefore this form of installation may not be
+      suitable for novice Linux users.</div>
+
+    <p>The <strong>graphical installation CD</strong> contains the NixOS
+      installer as well as X11, Plasma 5 Desktop and several applications. It’s a
+      <em>live CD</em>, so it allows you to get an impression of NixOS (and
+      the Nix package manager) before installing it.</p>
+
+    [% prefix = "https://channels.nixos.org/nixos-" _ latestNixOSSeries %]
+
+    <ul>
+      <li><a href="[%prefix%]/latest-nixos-graphical-x86_64-linux.iso">Graphical live CD, 64-bit Intel/AMD</a> (<a
+          href="[%prefix%]/latest-nixos-graphical-x86_64-linux.iso.sha256">SHA-256</a>)
+        <span class="label label-info">Recommended for most users</span></li>
+    </ul>
+
+    <p>The <strong>minimal installation CD</strong> does not contain X11,
+      and is therefore a lot smaller. You have to run the installer from
+      the console. It contains a number of rescue tools.</p>
+
+    <ul>
+      <li><a href="[%prefix%]/latest-nixos-minimal-x86_64-linux.iso">Minimal installation CD, 64-bit Intel/AMD</a> (<a
+          href="[%prefix%]/latest-nixos-minimal-x86_64-linux.iso.sha256">SHA-256</a>)</li>
+      <li><a href="[%prefix%]/latest-nixos-minimal-i686-linux.iso">Minimal installation CD, 32-bit Intel/AMD</a> (<a
+          href="[%prefix%]/latest-nixos-minimal-i686-linux.iso.sha256">SHA-256</a>)</li>
+    </ul>
+  </section>
+  <section class="tab-pane" id="nixos-virtualbox">
+    <p>This is a demo appliance for VirtualBox (in OVA format) that has
+      X11 and Plasma 5 enabled, as well as the VirtualBox guest additions. To
+      use it, download the OVA file, open VirtualBox, run “File → Import
+      Appliance” from the menu, select the OVA file, and click “Import”.
+      You can then start the virtual machine. When the KDE login screen
+      appears, you can log in as <strong>user <code>demo</code></strong>,
+      <strong>password <code>demo</code></strong>. To obtain a root shell, run
+      <code>sudo -i</code> in the KDE terminal (<code>konsole</code>).</p>
+
+    <ul>
+      <li><a href="[%prefix%]/latest-nixos-x86_64-linux.ova">VirtualBox appliance, 64-bit Intel/AMD</a> (<a
+          href="[%prefix%]/latest-nixos-x86_64-linux.ova.sha256">SHA-256</a>)</li>
+    </ul>
+  </section>
+  <section class="tab-pane" id="nixos-amazon">
+    <p>If you are an EC2 user, you can fire up a NixOS instance instantly
+      by using one of the AMIs listed below.</p>
+
+    <div class="row">
+      <table class="table">
```

Comment:
simplify (later). maybe just dropdown for region and one deploy button

the other information is duplicate. just mention it once in the text

Reply:
yeah, lets do this later.